WordPressAuthenticator/Services/WordPressComOAuthClientFacadeProtocol.swift

```diff
@@ -11,7 +11,7 @@ import WordPressKit
         password: String,
         multifactorCode: String?,
         success: @escaping (_ authToken: String?) -> Void,
-        needsMultifactor: ((_ userID: Int, _ nonceInfo: SocialLogin2FANonceInfo?) -> Void)?,
+        needsMultifactor: @escaping ((_ userID: Int, _ nonceInfo: SocialLogin2FANonceInfo?) -> Void),
```

Comment:
This API change makes is a breaking change, right?

Reply:
Yes indeed. I didn't add a breaking change entry in this PR, because [there is already one](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/7d32968c712fbfcbdb2c430b3b8f4faf4eb11c4b/CHANGELOG.md?plain=1#L37) that applies here too.